tests/python/utils.py

```diff
@@ -376,39 +376,44 @@ def nvfusertest_serde_check(test_fn: Callable):
     structure.
     """
     if disable_serde:
-        return test_fn
 
-    def inner_fn(*args, **kwargs):
-        self, fusion_func, inputs = args
-
-        # NOTE: For debug purposes, clear FusionCache before running first test
-        # so the behavior is more deterministic (PR #1848).
-        is_new_fusion_expected = kwargs.get("new_fusion_expected", True)
-        if debug_serde and is_new_fusion_expected:
-            FusionCache.reset()
-            assert FusionCache.get().num_fusions() == 0
-
-        # skip_serde_check is only used by the decorator so remove it before
-        # running test_fn
-        skip_serde_check = kwargs.pop("skip_serde_check", False)
-        if skip_serde_check:
+        def inner_fn(*args, **kwargs):
+            kwargs.pop("skip_serde_check")
+            return test_fn(*args, **kwargs)
+
+    else:
```

Comment:
nitpick: Isn't it cleaner to return `inner_fn` immediately if `disable_serde == True`?
e.g., use a short-circuit if-statement to avoid and else block.